submissions/rufusOtolorin/database/init.sql

```diff
@@ -0,0 +1,62 @@
+ -- TO CREATE DATABASE CALLED FOODBAG
+CREATE DATABASE foodbag;
+-- TO CONFIRM IF DATABASE IS CREATED
+SHOW DATABASES;
+ -- TO USE THE FOODBAG DATABASE
+USE foodbag;
+
+ -- TO START CREATING THE FOOD_BAG TABLE
+CREATE TABLE food (
+	id INT NOT NULL AUTO_INCREMENT,
+    type VARCHAR(255) NOT NULL,
+    PRIMARY KEY (id)
+);
+
+-- TO START CREATING THE ADMINS TABLE
+CREATE TABLE admins (
+	id INT NOT NULL AUTO_INCREMENT,
+    name VARCHAR(255) NOT NULL,
+    type INT NOT NULL,
+    email_address VARCHAR(255) NOT NULL,
+    phone_number VARCHAR(100) NOT NULL,
+    password VARCHAR(255) NOT NULL,
+    PRIMARY KEY (id)
+);
+
+-- TO START MAKING THE VENDOR_FOOD TABLE
+CREATE TABLE vendor_food (
+	id INT NOT NULL AUTO_INCREMENT,
+    admin_id INT NOT NULL,
+    food_id INT NOT NULL,
+    name VARCHAR(255) NOT NULL,
+    amount DECIMAL(10,2) NOT NULL,
+    PRIMARY KEY (id),
+    FOREIGN KEY (admin_id) REFERENCES admins (id),
+    FOREIGN KEY (food_id) REFERENCES food (id)
+);
+
+-- TO START CREATING CUSTOMERS ACCOUNT TABLE
+CREATE TABLE customers_account (
+	id INT NOT NULL AUTO_INCREMENT,
+    admin_id INT NOT NULL,
+    name VARCHAR(255) NOT NULL,
+    phone_number VARCHAR(100) NOT NULL,
+    address VARCHAR(255) NOT NULL,
+    payment_method VARCHAR(255),
```

Comment:
payment_method, etc as seen on the payment verification page. The first four columns mentioned are required and the other columns regarding payment are optional.
Change line 45 to:   payment_method ENUM('card', 'cash', 'transfer') NULL DEFAULT 'card'